js/renovation/ui/scroll_view/scrollable.tsx

```diff
@@ -67,7 +68,9 @@ export const viewFunction = (viewModel: Scrollable): JSX.Element => {
   );
 };
 
-type ScrollablePropsType = ScrollableProps & Pick<BaseWidgetProps, 'rtlEnabled' | 'disabled' | 'width' | 'height'>;
+type ScrollablePropsType = ScrollableProps
+& Pick<WidgetProps, 'aria'>
```

Comment:
Why it is declared in props type?
"this.setAria('role', 'form', this.$element());" - it looks like a const